local/bin/py/build/actions/integrations.py

```diff
@@ -905,6 +923,20 @@ def add_integration_frontmatter(
                 item["draft"] = not item.get("is_public", False)
                 item["integration_id"] = item.get("integration_id", integration_id)
                 item["integration_version"] = item.get("integration_version", integration_version)
+                # if tile.resources exists, add it to the front matter as further reading
+                further_reading = []
+                if item.get("tile", {}).get("resources"):
+                    for resource in item.get("tile", {}).get("resources"):
```

Comment:
nit: could remove the extra if block
```suggestion
                for resource in item.get("tile", {}).get("resources", []):
```